packages/back-end/src/models/InformationSchemaTablesModel.ts

```diff
@@ -0,0 +1,66 @@
+import omit from "lodash/omit";
+import z from "zod";
+import mongoose from "mongoose";
+import { InformationSchemaTablesInterface } from "../types/Integration";
+import { errorStringFromZodResult } from "../util/validation";
+import { logger } from "../util/logger";
+
+const informationSchemaTablesSchema = new mongoose.Schema({
+  organization: {
+    type: String,
+    index: true,
+  },
```

Comment:
I think we need an `id` field here to uniquely identify this document.  Table name is not unique enough